bin/heroics-generate

```diff
@@ -26,15 +30,36 @@ option_parser = OptionParser.new do |opts|
 end
 
 option_parser.parse!
-if ARGV.length != 3
-  puts option_parser
-else
-  module_name, schema_filepath, url = ARGV
-  schema = Heroics::Schema.new(MultiJson.decode(open(schema_filepath).read))
-  cache = 'Moneta.new(:Memory)'
-  if options[:cache_path]
-    cache = "Moneta.new(:File, dir: \"#{options[:cache_path]}\")"
+
+case ARGV.length
+when 1
+  config_file = ARGV[0]
+  if File.readable?(config_file)
```

Comment:
It might be nice to have a broader rescue somewhere to catch when a config file has syntax errors